Warning. sybase_connect

zerojano opened the pull request:
Sybase Connect Server message, Changed database context.
Although it is only a warning, it generates an error in the log with connections to external Sybase databases.
The error message is generated in moodle login, with system debugging enabled.

mnewnham commented:
Thanks for that patch. @dregad There seems no reason that we can't just merge this immediately

dregad commented:
@zerojano I'm fine with merging this, but before I do I was wondering if the same fix should also be applied to *connect()* method ?

Can you please clarify the exact error message you're getting, and the context to trigger it (ideally a code sample to reproduce)

zerojano commented:
@dregad I modified the connection method with Sybase, apparently there is where the error is generated, it is necessary to rescue that when modifying the debugging options> normal, the error is generated.

**Error message:**

https://example.moodle.cl/login/index.php?authldap_skipntlmsso=1

Warning: sybase_connect (): Sybase: Server message: Changed database context to 'webApplication'. (severity 10, procedure N / A) in /var/www/html/moodle/lib/adodb/drivers/adodb-sybase.inc.php on line 138
Skip to main content

× This page should redirect automatically. If nothing happens, please use the follow link below.
(Continue)

Output error, disable automatic redirection.

dregad commented:
I'm not sure to understand what you mean, maybe something got lost in translation ? 

As far as I can tell, you're not answering my question, which was - should the same fix also be applied to *connect()* method ?

zerojano commented:
@dregad I failed the translation.
Yes. it is necessary to modify both _connect () and _pconnect () methods,

Greetings.
